src/zeroconf/_services/info.py

```diff
@@ -519,6 +519,14 @@ def dns_addresses(
         self,
         override_ttl: Optional[int] = None,
         version: IPVersion = IPVersion.All,
+    ) -> List[DNSAddress]:
+        """Return matching DNSAddress from ServiceInfo."""
+        return self._dns_addresses(override_ttl, version)
```

Comment:
I would have preferred not to add the _ functions but I don't want this to be a breaking change